cmd/kubelet/app/server.go

```diff
@@ -1208,8 +1208,8 @@ func RunKubelet(kubeServer *options.KubeletServer, kubeDeps *kubelet.Dependencie
 	// Setup event recorder if required.
 	makeEventRecorder(kubeDeps, nodeName)
 
-	nodeIPs, err := nodeutil.ParseNodeIPArgument(kubeServer.NodeIP, kubeServer.CloudProvider, utilfeature.DefaultFeatureGate.Enabled(features.CloudDualStackNodeIPs))
-	if err != nil {
+	nodeIPs, invalidNodeIps, err := nodeutil.ParseNodeIPArgument(kubeServer.NodeIP, kubeServer.CloudProvider, utilfeature.DefaultFeatureGate.Enabled(features.CloudDualStackNodeIPs))
+	if len(invalidNodeIps) != 0 || err != nil {
```

Comment:
`len(invalidNodeIps) != 0` is not an error. Previously, those were logged and the kubelet continued. We need to replicate that behavior here, just with a single log call for all invalid (or rather, unparseable)  IPs.

Reply:
Sorry, I had implemented it without realizing the mistake. I have now corrected it so that logs are output for each invalid IP and the kubelet continues to operate, both in the same manner as in the original implementation.